packages/core/src/app/address/googleAutocomplete/AddressSelectorUk.ts

```diff
@@ -2,10 +2,14 @@ import AddressSelector from './AddressSelector';
 
 export default class AddressSelectorUK extends AddressSelector {
     getState(): string {
-        return this._get('administrative_area_level_2', 'long_name');
+        return '';
```

Comment:
Do we need to change this?

Reply:
Yes, because the ticket wants no more auto fill for UK's `state/province`.